corpus/expressions.txt

```diff
@@ -378,7 +378,7 @@ yield from a
       (expression_list (identifier))
       (yield (expression_list (integer)))))
   (expression_statement
-    (yield (identifier))))
+    (yield (expression_statement (identifier)))))
```

Comment:
You also can't have nested yields as far as I can tell. In both python 2 and 3, this parses fine:

```python
def foo():
  yield 5
```

but this does not:

```python
def foo():
  yield yield 1
           ^
SyntaxError: invalid syntax
```

This jives with the python grammar. I did notice that in python 3, you can put a yield inside of parentheses, like this:

```python
def foo():
  (yield 1) + (yield 2)
```

Is that the problem that you were originally hitting? It seems like the solution there is to add a `parenthesized_expression` rule that consists of parentheses containing exactly one expression (or a yield).